addons/binding/org.openhab.binding.zoneminder/src/main/java/org/openhab/binding/zoneminder/handler/ZoneMinderThingMonitorHandler.java

```diff
@@ -782,14 +816,17 @@ protected void onFetchData() {
                     channelDaemonFrame = false;
                 }
             }
-        } finally {
+        } finally
+
```

Comment:
By "Revert this change" you mean don't logError? I have changed some logic as well, those changes is relevant in my opinion